### PR TITLE
Adding Hindsight module.

### PR DESCRIPTION
**Pull request by hed-fb**
Adding hindsight (https://github.com/obsidianforensics/hindsight) module to parse Chromium profile collections with Turbinia.

### CODE REVIEW

**Comment by ericzinnikas** [comment]
Will talk to Hasan, I think some merge/rebase stuff messed this PR up a bit.

File: turbinia/workers/hindsight.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    # Create a path that we can write the new file to.
+    output_file_path = os.path.join(
+        self.output_dir, '{0:s}'.format(self.id))
+    output_evidence.local_path = output_file_path
+
+    # Create a path for the log file
+    hindsight_log = os.path.join(self.output_dir, '{0:s}.log'.format(self.id))
+
+    cmd = ('hindsight.py -i {0:s} --format={1:s} --browser_type={2:s} '
```
Comment (aarontp) [inline turbinia/workers/hindsight.py]: We should also add `pyhindsight` into the setup/requirements files as a worker dependency.

Comment (hed-fb) [reply turbinia/workers/hindsight.py]: Thanks for your feedback, Aaron! Will resolve the issues as soon as I get the chance.

**Comment by ericzinnikas** [comment]
@aarontp thoughts on using `click` instead of `argparse`? I've only recently found out about it and started using it in some projects. A lot more powerful (and simpler) imo, for building CLI tools: https://click.palletsprojects.com/en/7.x/

Maybe worth considering as `turbiniactl` gets more complicated. It is really easy to set argument defaults, prompt options, etc.

**Comment by aarontp** [comment]
@ericzinnikas Yeah, I'd be open to changing to something like that.  I don't know a lot about it, but I think Timesketch uses this as well.

Another thing I'd like to solve with the command args is: https://github.com/google/turbinia/issues/41

**Pull request closed without merging**